qutebrowser/misc/sessions.py

```diff
@@ -179,6 +181,8 @@ def _save_tab_item(self, tab, idx, item):
             # QtWebEngine
             user_data = None
 
+        data['last_visited'] = item.lastVisited().toString()
```

Comment:
The default `Qt::TextDate` format is locale-dependent, so `Qt::ISODate` would probably be a better fit here.